<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OneWire</class>
 <widget class="QWidget" name="OneWire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Write Comand:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QPushButton" name="button_search">
       <property name="text">
        <string>Search Bus</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QPushButton" name="button_reset">
       <property name="text">
        <string>Reset Bus</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QPushButton" name="button_read_byte">
       <property name="text">
        <string>Read Byte</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Write Byte:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="SpinBoxHex" name="spinbox_write_command">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="combo_box_write_command">
       <item>
        <property name="text">
         <string>Skip ID</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="button_write_byte">
       <property name="text">
        <string>Write</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="button_write_command">
       <property name="text">
        <string>Write</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="SpinBoxHex" name="spinbox_write_byte">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="tree_widget">
     <column>
      <property name="text">
       <string>Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Command</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpinBoxHex</class>
   <extends>QSpinBox</extends>
   <header location="global">brickv.spin_box_hex</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
